include/RAJA/pattern/launch/launch_core.hpp

```diff
@@ -37,11 +40,9 @@
 namespace RAJA
 {
 
-namespace expt
-{
-
 // GPU or CPU threads available
-enum ExecPlace { HOST, DEVICE, NUM_PLACES };
+//strongly type the ExecPlace (guards agaist errors)
+enum struct ExecPlace : int { HOST, DEVICE, NUM_PLACES };
```

Comment:
What is NUM_PLACES used for?

Reply:
I think it's a "count" data member. If kept at the end of the enum, it will always be equal to the number of exec places. Probably used as a loop bound or check in the code.

Reply:
Ahh, fair enough.